### ISSUE TITLE
Remove Support For s390x

### ISSUE THREAD

**Issue by dave-tucker** [linuxkit/linuxkit#3676]
Linuxkit has supported s390x for a while, but we are not aware of any users of this architecture.
While IBM have graciously helped provide resources for both the initial port and providing machines for build/test we are but a small group of volunteers and the additional overhead of supporting an architecture, kernel/package builds etc... is hard to justify without any users.

As such I propose that we remove support for s390x for now. This is not forever, and we'll be happy to open discussions about bringing it back in future if there are users that want it. For now though we have to prioritse where our maintainers can spend their (limited) time.

/cc @justincormack @deitch @rn @djs55 as i'd like you to ack this

**Comment by thaJeztah** [comment]
makes sense to me

**Comment by deitch** [comment]
Agreed.

**Comment by djs55** [comment]
OK!

**Comment by justincormack** [comment]
SGTM

**Comment by justincormack** [comment]
Note that things should still work on s390x, this mainly means no prebuilt binaries for now.

**Comment by rn** [comment]
I agree with Justin. Happy to remove it from the builds and as a default arch from `linuxkit` but would be good to keep support where needed to build binaries and packages. This does imply, though removing it from `linuxkit/alpine` and the kernel builds, which are special.

FWIW, I currently have some changes to kernel builds etc which I'm still debugging and I'd appreciate if I don't get a conflicting PR while doing so :)
